static/src/stylesheets/module/tailor/_tailor-survey-overlay.scss

```diff
@@ -0,0 +1,128 @@
+.tailor-survey-overlay,
+.tailor-survey-background {
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	z-index: 1050;
+	position: fixed;
+	display: block;
+}
+
+.tailor-survey-overlay .tailor-survey-background {
+	background: #000000;
+	opacity: .5;
+	z-index: 1;
+}
+
+.tailor-survey-overlay .tailor-survey-container {
+	width: 380px;
+	height: 300px;
+	position: absolute;
+	margin: auto;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	background-color: $guardian-brand;
+	z-index: 2;
+
+	.tailor-survey-container-header,
+	.tailor-survey-proceed-button {
+		font-family: $f-sans-serif-text;
+		font-weight: 900;
+		font-size: 18px;
+	}
+
+	.tailor-survey-container-header {
+		height: 42px;
+		width: 100%;
+		background-color: $news-support-1;
+		color: $neutral-1;
+		line-height: 42px;
+		padding-left: 8px;
+		box-sizing: border-box;
+
+		.inline-marque-36 {
+			position: absolute;
+			top: 3px;
+			right: 8px;
+		}
+	}
+
+	h1,
+	p {
+		color: $neutral-6;
+		font-size: 22px;
+		line-height: 25px;
+		margin: 12px 8px;
+		font-family: $f-serif-headline;
+	}
+
+	h1 {
```

Comment:
The font-weight and family can probably be bought in using the font mixins in `stylesheets/pasteup/typography/_src.scss` -  `@fs-headline`